[2025-07-29 15:00:20,009: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-29 15:00:20,014: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-29 15:02:07,961: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-29 15:02:07,964: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-29 15:02:34,714: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-29 15:02:34,717: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-29 15:02:53,851: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-29 15:02:53,889: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-29 15:02:53,897: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-29 15:02:53,899: INFO: common: Created directory at: artifacts]
[2025-07-29 15:02:53,901: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-07-29 15:02:55,326: INFO: 760908179: Downloaded file: artifacts/data_ingestion/data.zip with following info: Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: C2B8:A6F19:7D83FD:8F970B:6888802B
Accept-Ranges: bytes
Date: Tue, 29 Jul 2025 08:02:53 GMT
Via: 1.1 varnish
X-Served-By: cache-sin-wsss1830084-SIN
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1753776173.248094,VS0,VE529
Vary: Authorization,Accept-Encoding
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 6d311221247caeaef51c1191e1dd3e223fec4fc4
Expires: Tue, 29 Jul 2025 08:07:53 GMT
Source-Age: 0

]
[2025-07-29 15:02:55,425: INFO: 760908179: Extracted zip file to : artifacts/data_ingestion]
[2025-07-30 22:26:21,265: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-07-30 22:26:21,281: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-30 22:26:21,298: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-30 22:26:21,304: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-30 22:26:21,306: INFO: common: Created directory at: artifacts]
[2025-07-30 22:26:21,307: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-07-30 22:26:23,058: INFO: data_ingestion: Downloaded file: artifacts/data_ingestion/data.zip with following info: Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 97EF:346FD:251202:292ADA:688A399F
Accept-Ranges: bytes
Date: Wed, 30 Jul 2025 15:26:24 GMT
Via: 1.1 varnish
X-Served-By: cache-sin-wsss1830043-SIN
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1753889183.138416,VS0,VE989
Vary: Authorization,Accept-Encoding
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 4e41527de371b4ec771389ef2082bc51511f6da8
Expires: Wed, 30 Jul 2025 15:31:24 GMT
Source-Age: 0

]
[2025-07-30 22:26:23,146: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-07-30 22:26:23,146: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-07-30 23:23:18,787: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-30 23:23:18,791: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-30 23:23:18,820: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-30 23:23:18,822: INFO: common: Created directory at: artifacts]
[2025-07-30 23:23:35,118: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-30 23:23:35,120: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-30 23:23:35,130: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-30 23:23:35,133: INFO: common: Created directory at: artifacts]
[2025-07-30 23:23:35,135: INFO: common: Created directory at: artifacts/data_validation]
[2025-07-30 23:38:41,043: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-07-30 23:38:41,678: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-07-30 23:38:41,684: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-30 23:38:41,686: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-30 23:38:41,695: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-30 23:38:41,698: INFO: common: Created directory at: artifacts]
[2025-07-30 23:38:41,705: ERROR: main: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config']
Traceback (most recent call last):
  File "c:\Users\admin\OneDrive\Dokumen\Portofolio\Data Sceince (Python)\End-to-end\main.py", line 10, in <module>
    obj.main()
    ~~~~~~~~^^
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\pipeline\stage_01_data_ingestion.py", line 13, in main
    data_ingestion_config = config.get_data_ingestion_config()
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config'. Did you mean: 'get_data_validation_config'?
[2025-07-30 23:44:17,826: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-07-30 23:44:18,139: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-07-30 23:44:18,149: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-30 23:44:18,150: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-30 23:44:18,155: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-30 23:44:18,157: INFO: common: Created directory at: artifacts]
[2025-07-30 23:44:18,158: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-07-30 23:44:19,277: INFO: data_ingestion: Downloaded file: artifacts/data_ingestion/data.zip with following info: Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: E7D6:2B1519:3746D7:3D4154:688A4BE3
Accept-Ranges: bytes
Date: Wed, 30 Jul 2025 16:44:20 GMT
Via: 1.1 varnish
X-Served-By: cache-sin-wsss1830027-SIN
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1753893860.952536,VS0,VE406
Vary: Authorization,Accept-Encoding
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 82120c7228cdc144dc349fe89d3482704c408f53
Expires: Wed, 30 Jul 2025 16:49:20 GMT
Source-Age: 0

]
[2025-07-30 23:44:19,312: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-07-30 23:44:19,314: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-07-30 23:44:19,315: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-07-30 23:44:19,319: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-30 23:44:19,321: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-30 23:44:19,328: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-30 23:44:19,332: INFO: common: Created directory at: artifacts]
[2025-07-30 23:44:19,333: INFO: common: Created directory at: artifacts/data_validation]
[2025-07-30 23:44:19,334: ERROR: main: 'DataValidation' object has no attribute 'validate_data']
Traceback (most recent call last):
  File "C:\Users\admin\OneDrive\Dokumen\Portofolio\Data Sceince (Python)\End-to-end\main.py", line 20, in <module>
    obj.main()
    ~~~~~~~~^^
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\pipeline\stage_02_data_validation.py", line 15, in main
    data_validation.validate_data()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'DataValidation' object has no attribute 'validate_data'
[2025-07-30 23:47:08,804: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-07-30 23:47:09,119: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-07-30 23:47:09,121: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-30 23:47:09,122: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-30 23:47:09,124: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-30 23:47:09,124: INFO: common: Created directory at: artifacts]
[2025-07-30 23:47:09,125: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-07-30 23:47:09,757: INFO: data_ingestion: Downloaded file: artifacts/data_ingestion/data.zip with following info: Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: E7D6:2B1519:3746D7:3D4154:688A4BE3
Accept-Ranges: bytes
Date: Wed, 30 Jul 2025 16:47:10 GMT
Via: 1.1 varnish
X-Served-By: cache-sin-wsss1830068-SIN
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1753894031.880529,VS0,VE1
Vary: Authorization,Accept-Encoding
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 5e013c06d0a2f27a43809755fa7fb00dd777cfa8
Expires: Wed, 30 Jul 2025 16:52:10 GMT
Source-Age: 171

]
[2025-07-30 23:47:09,794: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-07-30 23:47:09,797: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-07-30 23:47:09,798: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-07-30 23:47:09,805: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-30 23:47:09,808: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-30 23:47:09,815: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-30 23:47:09,816: INFO: common: Created directory at: artifacts]
[2025-07-30 23:47:09,817: INFO: common: Created directory at: artifacts/data_validation]
[2025-07-30 23:47:09,818: ERROR: main: 'DataValidation' object has no attribute 'validate_all_data']
Traceback (most recent call last):
  File "C:\Users\admin\OneDrive\Dokumen\Portofolio\Data Sceince (Python)\End-to-end\main.py", line 20, in <module>
    obj.main()
    ~~~~~~~~^^
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\pipeline\stage_02_data_validation.py", line 15, in main
    data_validation.validate_all_data()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'DataValidation' object has no attribute 'validate_all_data'
[2025-07-30 23:47:36,484: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-07-30 23:47:36,808: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-07-30 23:47:36,809: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-30 23:47:36,810: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-30 23:47:36,812: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-30 23:47:36,813: INFO: common: Created directory at: artifacts]
[2025-07-30 23:47:36,813: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-07-30 23:47:37,029: INFO: data_ingestion: Downloaded file: artifacts/data_ingestion/data.zip with following info: Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: E7D6:2B1519:3746D7:3D4154:688A4BE3
Accept-Ranges: bytes
Date: Wed, 30 Jul 2025 16:47:38 GMT
Via: 1.1 varnish
X-Served-By: cache-sin-wsss1830062-SIN
X-Cache: HIT
X-Cache-Hits: 1
X-Timer: S1753894058.146848,VS0,VE3
Vary: Authorization,Accept-Encoding
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 451ad94a1d939abdea326782a42ce3e777fc1292
Expires: Wed, 30 Jul 2025 16:52:38 GMT
Source-Age: 198

]
[2025-07-30 23:47:37,102: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-07-30 23:47:37,102: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-07-30 23:47:37,103: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-07-30 23:47:37,106: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-30 23:47:37,110: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-30 23:47:37,117: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-30 23:47:37,120: INFO: common: Created directory at: artifacts]
[2025-07-30 23:47:37,121: INFO: common: Created directory at: artifacts/data_validation]
[2025-07-30 23:47:37,167: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-07-31 21:25:25,285: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-07-31 21:32:36,215: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-31 21:32:36,218: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-31 21:32:36,226: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-31 21:32:36,230: INFO: common: Created directory at: artifacts]
[2025-07-31 21:32:36,232: INFO: common: Created directory at: artifacts/data_transformation]
[2025-07-31 21:32:36,325: INFO: 838343783: Train and test data saved at artifacts\data_transformation]
[2025-07-31 21:32:36,326: INFO: 838343783: Train data shape: (1199, 12), Test data shape: (400, 12)]
[2025-07-31 21:43:29,851: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-07-31 21:43:31,432: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-07-31 21:43:31,434: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-31 21:43:31,435: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-31 21:43:31,437: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-31 21:43:31,438: INFO: common: Created directory at: artifacts]
[2025-07-31 21:43:31,438: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-07-31 21:43:32,578: INFO: data_ingestion: Downloaded file: artifacts/data_ingestion/data.zip with following info: Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: FDC0:312B49:B6DD4B:D180B1:688B8113
Accept-Ranges: bytes
Date: Thu, 31 Jul 2025 14:43:33 GMT
Via: 1.1 varnish
X-Served-By: cache-sin-wsss1830024-SIN
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1753973013.653446,VS0,VE373
Vary: Authorization,Accept-Encoding
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: ed63d7added0626c6f22f2f1482deb7341406cc6
Expires: Thu, 31 Jul 2025 14:48:33 GMT
Source-Age: 0

]
[2025-07-31 21:43:32,630: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-07-31 21:43:32,632: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-07-31 21:43:32,634: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-07-31 21:43:32,643: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-31 21:43:32,645: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-31 21:43:32,651: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-31 21:43:32,655: INFO: common: Created directory at: artifacts]
[2025-07-31 21:43:32,660: INFO: common: Created directory at: artifacts/data_validation]
[2025-07-31 21:43:32,699: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-07-31 21:43:32,699: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-07-31 21:43:32,703: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-31 21:43:32,706: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-31 21:43:32,712: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-31 21:43:32,714: INFO: common: Created directory at: artifacts]
[2025-07-31 21:43:32,716: INFO: common: Created directory at: artifacts/data_transformation]
[2025-07-31 21:43:32,753: INFO: data_transformation: Train and test data saved at artifacts\data_transformation]
[2025-07-31 21:43:32,753: INFO: data_transformation: Train data shape: (1199, 12), Test data shape: (400, 12)]
[2025-07-31 21:43:32,754: INFO: main: >>>>>> stage Data Transformation Stage completed <<<<<<]
[2025-07-31 22:15:42,213: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-07-31 22:15:45,877: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-07-31 22:15:45,891: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-31 22:15:45,893: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-31 22:15:45,896: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-31 22:15:45,898: INFO: common: Created directory at: artifacts]
[2025-07-31 22:15:45,899: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-07-31 22:15:47,036: INFO: data_ingestion: Downloaded file: artifacts/data_ingestion/data.zip with following info: Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: FDC0:312B49:B6DD4B:D180B1:688B8113
Accept-Ranges: bytes
Date: Thu, 31 Jul 2025 15:15:47 GMT
Via: 1.1 varnish
X-Served-By: cache-sin-wsss1830077-SIN
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1753974947.218413,VS0,VE297
Vary: Authorization,Accept-Encoding
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 28c202c363dad0dfc2fd96a4fd45d948535ae0ca
Expires: Thu, 31 Jul 2025 15:20:47 GMT
Source-Age: 0

]
[2025-07-31 22:15:47,145: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-07-31 22:15:47,146: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-07-31 22:15:47,146: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-07-31 22:15:47,150: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-31 22:15:47,153: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-31 22:15:47,163: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-31 22:15:47,164: INFO: common: Created directory at: artifacts]
[2025-07-31 22:15:47,165: INFO: common: Created directory at: artifacts/data_validation]
[2025-07-31 22:15:47,212: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-07-31 22:15:47,213: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-07-31 22:15:47,220: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-31 22:15:47,225: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-31 22:15:47,227: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-31 22:15:47,229: INFO: common: Created directory at: artifacts]
[2025-07-31 22:15:47,230: INFO: common: Created directory at: artifacts/data_transformation]
[2025-07-31 22:15:47,291: INFO: data_transformation: Train and test data saved at artifacts\data_transformation]
[2025-07-31 22:15:47,292: INFO: data_transformation: Train data shape: (1199, 12), Test data shape: (400, 12)]
[2025-07-31 22:15:47,293: INFO: main: >>>>>> stage Data Transformation Stage completed <<<<<<]
[2025-07-31 22:15:47,293: INFO: main: >>>>>> stage Model Training Stage started <<<<<<]
[2025-07-31 22:15:47,298: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-31 22:15:47,300: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-31 22:15:47,305: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-31 22:15:47,307: INFO: common: Created directory at: artifacts]
[2025-07-31 22:15:47,307: INFO: common: Created directory at: artifacts/data_transformation]
[2025-07-31 22:15:47,344: INFO: data_transformation: Train and test data saved at artifacts\data_transformation]
[2025-07-31 22:15:47,344: INFO: data_transformation: Train data shape: (1199, 12), Test data shape: (400, 12)]
[2025-07-31 22:15:47,344: INFO: main: >>>>>> stage Model Training Stage completed <<<<<<]
[2025-07-31 22:17:25,093: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-07-31 22:17:26,759: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-07-31 22:17:26,768: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-31 22:17:26,770: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-31 22:17:26,775: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-31 22:17:26,776: INFO: common: Created directory at: artifacts]
[2025-07-31 22:17:26,777: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-07-31 22:17:27,021: INFO: data_ingestion: Downloaded file: artifacts/data_ingestion/data.zip with following info: Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: FDC0:312B49:B6DD4B:D180B1:688B8113
Accept-Ranges: bytes
Date: Thu, 31 Jul 2025 15:17:27 GMT
Via: 1.1 varnish
X-Served-By: cache-sin-wsss1830089-SIN
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1753975048.518102,VS0,VE3
Vary: Authorization,Accept-Encoding
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 200f7dfdc1f4bea894df52f2a80101e91a6687b3
Expires: Thu, 31 Jul 2025 15:22:27 GMT
Source-Age: 100

]
[2025-07-31 22:17:27,092: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-07-31 22:17:27,094: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-07-31 22:17:27,097: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-07-31 22:17:27,103: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-31 22:17:27,105: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-31 22:17:27,113: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-31 22:17:27,115: INFO: common: Created directory at: artifacts]
[2025-07-31 22:17:27,116: INFO: common: Created directory at: artifacts/data_validation]
[2025-07-31 22:17:27,153: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-07-31 22:17:27,153: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-07-31 22:17:27,160: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-31 22:17:27,163: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-31 22:17:27,165: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-31 22:17:27,166: INFO: common: Created directory at: artifacts]
[2025-07-31 22:17:27,167: INFO: common: Created directory at: artifacts/data_transformation]
[2025-07-31 22:17:27,203: INFO: data_transformation: Train and test data saved at artifacts\data_transformation]
[2025-07-31 22:17:27,203: INFO: data_transformation: Train data shape: (1199, 12), Test data shape: (400, 12)]
[2025-07-31 22:17:27,204: INFO: main: >>>>>> stage Data Transformation Stage completed <<<<<<]
[2025-07-31 22:17:27,204: INFO: main: >>>>>> stage Model Training Stage started <<<<<<]
[2025-07-31 22:17:27,209: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-07-31 22:17:27,211: INFO: common: YAML file: params.yaml loaded successfully]
[2025-07-31 22:17:27,214: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-07-31 22:17:27,215: INFO: common: Created directory at: artifacts]
[2025-07-31 22:17:27,216: INFO: common: Created directory at: artifacts/model_training]
[2025-07-31 22:17:27,303: INFO: main: >>>>>> stage Model Training Stage completed <<<<<<]
[2025-08-02 10:54:04,342: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-08-02 22:35:31,748: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-02 22:35:31,786: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-02 22:35:31,814: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-02 22:35:31,818: INFO: common: Created directory at: artifacts/model_evaluation]
[2025-08-02 22:35:33,098: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-02 22:35:33,113: INFO: helpers: Accessing as rajwaAth]
[2025-08-02 22:35:34,247: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/rajwaAth/MLOps-Project "HTTP/1.1 200 OK"]
[2025-08-02 22:35:35,199: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-02 22:35:35,199: INFO: helpers: Initialized MLflow to track repo "rajwaAth/MLOps-Project"]
[2025-08-02 22:35:35,215: INFO: helpers: Repository rajwaAth/MLOps-Project initialized!]
[2025-08-02 22:36:22,800: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-02 22:36:22,804: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-02 22:36:22,811: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-02 22:36:22,814: INFO: common: Created directory at: artifacts/model_evaluation]
[2025-08-02 22:36:24,788: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/rajwaAth/MLOps-Project "HTTP/1.1 200 OK"]
[2025-08-02 22:36:24,818: INFO: helpers: Initialized MLflow to track repo "rajwaAth/MLOps-Project"]
[2025-08-02 22:36:24,833: INFO: helpers: Repository rajwaAth/MLOps-Project initialized!]
[2025-08-02 22:36:24,943: INFO: common: JSON file saved at: artifacts\model_evaluation\metrics.json]
[2025-08-02 22:43:14,256: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-02 22:43:14,274: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-02 22:43:14,299: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-02 22:43:14,301: INFO: common: Created directory at: artifacts/model_evaluation]
[2025-08-02 22:43:15,229: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/rajwaAth/MLOps-Project "HTTP/1.1 200 OK"]
[2025-08-02 22:43:15,231: INFO: helpers: Initialized MLflow to track repo "rajwaAth/MLOps-Project"]
[2025-08-02 22:43:15,231: INFO: helpers: Repository rajwaAth/MLOps-Project initialized!]
[2025-08-02 22:43:15,294: INFO: common: JSON file saved at: artifacts\model_evaluation\metrics.json]
[2025-08-02 23:02:19,670: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-02 23:02:19,682: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-02 23:02:19,690: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-02 23:02:19,697: INFO: common: Created directory at: artifacts/model_evaluation]
[2025-08-02 23:02:20,664: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-02 23:02:20,664: INFO: helpers: Accessing as rajwaAth]
[2025-08-02 23:02:21,654: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/rajwaAth/MLOps-Project "HTTP/1.1 200 OK"]
[2025-08-02 23:02:22,696: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-02 23:02:22,706: INFO: helpers: Initialized MLflow to track repo "rajwaAth/MLOps-Project"]
[2025-08-02 23:02:22,714: INFO: helpers: Repository rajwaAth/MLOps-Project initialized!]
[2025-08-02 23:02:23,086: INFO: common: JSON file saved at: artifacts\model_evaluation\metrics.json]
[2025-08-02 23:15:07,858: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-02 23:15:07,866: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-02 23:15:07,871: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-02 23:15:07,875: INFO: common: Created directory at: artifacts/model_evaluation]
[2025-08-02 23:15:08,715: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/rajwaAth/MLOps-Project "HTTP/1.1 200 OK"]
[2025-08-02 23:15:08,740: INFO: helpers: Initialized MLflow to track repo "rajwaAth/MLOps-Project"]
[2025-08-02 23:15:08,744: INFO: helpers: Repository rajwaAth/MLOps-Project initialized!]
[2025-08-02 23:15:08,849: INFO: common: JSON file saved at: artifacts\model_evaluation\metrics.json]
[2025-08-02 23:22:57,036: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-08-02 23:22:57,046: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-02 23:22:57,056: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-02 23:22:57,056: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-02 23:22:57,056: INFO: common: Created directory at: artifacts]
[2025-08-02 23:22:57,056: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-08-02 23:22:58,792: INFO: data_ingestion: Downloaded file: artifacts/data_ingestion/data.zip with following info: Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 9C41:1660AB:FB5E8:33261E:688E3B62
Accept-Ranges: bytes
Date: Sat, 02 Aug 2025 16:22:59 GMT
Via: 1.1 varnish
X-Served-By: cache-qpg1220-QPG
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1754151779.393319,VS0,VE276
Vary: Authorization,Accept-Encoding
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 2f5ec3889305a9f0dd644a25f25b5638c994f90d
Expires: Sat, 02 Aug 2025 16:27:59 GMT
Source-Age: 0

]
[2025-08-02 23:22:58,853: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-08-02 23:22:58,856: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-08-02 23:22:58,856: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-08-02 23:22:58,864: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-02 23:22:58,865: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-02 23:22:58,868: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-02 23:22:58,869: INFO: common: Created directory at: artifacts]
[2025-08-02 23:22:58,872: INFO: common: Created directory at: artifacts/data_validation]
[2025-08-02 23:22:58,925: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-08-02 23:22:58,925: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-08-02 23:22:58,934: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-02 23:22:58,939: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-02 23:22:58,943: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-02 23:22:58,944: INFO: common: Created directory at: artifacts]
[2025-08-02 23:22:58,946: INFO: common: Created directory at: artifacts/data_transformation]
[2025-08-02 23:22:58,992: INFO: data_transformation: Train and test data saved at artifacts\data_transformation]
[2025-08-02 23:22:58,992: INFO: data_transformation: Train data shape: (1199, 12), Test data shape: (400, 12)]
[2025-08-02 23:22:58,993: INFO: main: >>>>>> stage Data Transformation Stage completed <<<<<<]
[2025-08-02 23:22:58,993: INFO: main: >>>>>> stage Model Training Stage started <<<<<<]
[2025-08-02 23:22:58,998: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-02 23:22:59,000: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-02 23:22:59,004: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-02 23:22:59,005: INFO: common: Created directory at: artifacts]
[2025-08-02 23:22:59,006: INFO: common: Created directory at: artifacts/model_training]
[2025-08-02 23:22:59,056: INFO: main: >>>>>> stage Model Training Stage completed <<<<<<]
[2025-08-02 23:22:59,057: INFO: main: >>>>>> stage Model Evaluation Stage started <<<<<<]
[2025-08-02 23:22:59,060: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-02 23:22:59,062: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-02 23:22:59,065: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-02 23:22:59,066: INFO: common: Created directory at: artifacts]
[2025-08-02 23:22:59,067: INFO: common: Created directory at: artifacts/model_evaluation]
[2025-08-02 23:22:59,747: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-02 23:22:59,761: INFO: helpers: Accessing as rajwaAth]
[2025-08-02 23:23:00,644: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/rajwaAth/MLOps-Project "HTTP/1.1 200 OK"]
[2025-08-02 23:23:01,352: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-02 23:23:01,352: INFO: helpers: Initialized MLflow to track repo "rajwaAth/MLOps-Project"]
[2025-08-02 23:23:01,352: INFO: helpers: Repository rajwaAth/MLOps-Project initialized!]
[2025-08-02 23:23:01,377: INFO: common: JSON file saved at: artifacts\model_evaluation\metrics.json]
[2025-08-02 23:23:22,517: INFO: main: >>>>>> stage Model Evaluation Stage completed <<<<<<]
[2025-08-02 23:29:51,761: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-08-02 23:29:51,765: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-02 23:29:51,788: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-02 23:29:51,791: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-02 23:29:51,792: INFO: common: Created directory at: artifacts]
[2025-08-02 23:29:51,793: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-08-02 23:29:51,793: ERROR: main: the return value (str) is not an instance of int]
Traceback (most recent call last):
  File "c:\Users\admin\OneDrive\Dokumen\Portofolio\Data Sceince (Python)\End-to-end\main.py", line 13, in <module>
    obj.main()
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\pipeline\stage_01_data_ingestion.py", line 15, in main
    data_ingestion.download_file()
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\components\data_ingestion.py", line 21, in download_file
    logger.info(f"File already exists of size: {get_size(Path(self.config.local_data_file))}")
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\utils\common.py", line 118, in get_size
    return f"{size_in_kb} KB"
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\typeguard\_functions.py", line 166, in check_return_type
    check_type_internal(retval, annotation, memo)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\typeguard\_checkers.py", line 965, in check_type_internal
    raise TypeCheckError(f"is not an instance of {qualified_name(origin_type)}")
typeguard.TypeCheckError: the return value (str) is not an instance of int
[2025-08-02 23:31:29,539: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-08-02 23:31:29,539: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-02 23:31:29,557: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-02 23:31:29,560: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-02 23:31:29,560: INFO: common: Created directory at: artifacts]
[2025-08-02 23:31:29,561: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-08-02 23:31:29,561: ERROR: main: the return value (str) is not an instance of int]
Traceback (most recent call last):
  File "c:\Users\admin\OneDrive\Dokumen\Portofolio\Data Sceince (Python)\End-to-end\main.py", line 13, in <module>
    obj.main()
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\pipeline\stage_01_data_ingestion.py", line 15, in main
    data_ingestion.download_file()
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\components\data_ingestion.py", line 21, in download_file
    logger.info(f"File already exists of size: {get_size(Path(self.config.local_data_file))}")
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\utils\common.py", line 118, in get_size
    return f"{size_in_kb} KB"
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\typeguard\_functions.py", line 166, in check_return_type
    check_type_internal(retval, annotation, memo)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\typeguard\_checkers.py", line 965, in check_type_internal
    raise TypeCheckError(f"is not an instance of {qualified_name(origin_type)}")
typeguard.TypeCheckError: the return value (str) is not an instance of int
[2025-08-02 23:35:57,605: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-08-02 23:35:57,605: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-02 23:35:57,620: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-02 23:35:57,626: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-02 23:35:57,629: INFO: common: Created directory at: artifacts]
[2025-08-02 23:35:57,630: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-08-02 23:35:57,632: INFO: data_ingestion: File already exists of size: 22.7822265625 KB]
[2025-08-02 23:35:57,638: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-08-02 23:35:57,638: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-08-02 23:35:57,638: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-08-02 23:35:57,653: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-02 23:35:57,653: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-02 23:35:57,670: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-02 23:35:57,670: INFO: common: Created directory at: artifacts]
[2025-08-02 23:35:57,670: INFO: common: Created directory at: artifacts/data_validation]
[2025-08-02 23:35:57,753: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-08-02 23:35:57,753: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-08-02 23:35:57,775: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-02 23:35:57,776: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-02 23:35:57,777: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-02 23:35:57,777: INFO: common: Created directory at: artifacts]
[2025-08-02 23:35:57,782: INFO: common: Created directory at: artifacts/data_transformation]
[2025-08-02 23:35:57,823: INFO: data_transformation: Train and test data saved at artifacts\data_transformation]
[2025-08-02 23:35:57,823: INFO: data_transformation: Train data shape: (1199, 12), Test data shape: (400, 12)]
[2025-08-02 23:35:57,823: INFO: main: >>>>>> stage Data Transformation Stage completed <<<<<<]
[2025-08-02 23:35:57,823: INFO: main: >>>>>> stage Model Training Stage started <<<<<<]
[2025-08-02 23:35:57,841: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-02 23:35:57,842: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-02 23:35:57,851: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-02 23:35:57,852: INFO: common: Created directory at: artifacts]
[2025-08-02 23:35:57,853: INFO: common: Created directory at: artifacts/model_training]
[2025-08-02 23:35:57,905: INFO: main: >>>>>> stage Model Training Stage completed <<<<<<]
[2025-08-02 23:35:57,919: INFO: main: >>>>>> stage Model Evaluation Stage started <<<<<<]
[2025-08-02 23:35:57,927: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-02 23:35:57,930: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-02 23:35:57,936: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-02 23:35:57,938: INFO: common: Created directory at: artifacts]
[2025-08-02 23:35:57,943: INFO: common: Created directory at: artifacts/model_evaluation]
[2025-08-02 23:35:58,653: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-02 23:35:58,669: INFO: helpers: Accessing as rajwaAth]
[2025-08-02 23:35:59,436: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/rajwaAth/MLOps-Project "HTTP/1.1 200 OK"]
[2025-08-02 23:36:00,203: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-02 23:36:00,203: INFO: helpers: Initialized MLflow to track repo "rajwaAth/MLOps-Project"]
[2025-08-02 23:36:00,203: INFO: helpers: Repository rajwaAth/MLOps-Project initialized!]
[2025-08-02 23:36:00,260: INFO: common: JSON file saved at: artifacts\model_evaluation\metrics.json]
[2025-08-02 23:36:22,640: INFO: main: >>>>>> stage Model Evaluation Stage completed <<<<<<]
[2025-08-04 13:39:51,929: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-08-04 13:39:51,956: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-04 13:39:51,973: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-04 13:39:51,990: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-04 13:39:51,992: INFO: common: Created directory at: artifacts]
[2025-08-04 13:39:51,993: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-08-04 13:39:51,993: INFO: data_ingestion: File already exists of size: 22.7822265625 KB]
[2025-08-04 13:39:52,034: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-08-04 13:39:52,034: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-08-04 13:39:52,034: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-08-04 13:39:52,040: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-04 13:39:52,043: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-04 13:39:52,043: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-04 13:39:52,046: INFO: common: Created directory at: artifacts]
[2025-08-04 13:39:52,046: INFO: common: Created directory at: artifacts/data_validation]
[2025-08-04 13:39:52,098: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-08-04 13:39:52,098: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-08-04 13:39:52,108: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-04 13:39:52,111: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-04 13:39:52,120: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-04 13:39:52,123: INFO: common: Created directory at: artifacts]
[2025-08-04 13:39:52,124: INFO: common: Created directory at: artifacts/data_transformation]
[2025-08-04 13:39:52,188: INFO: data_transformation: Train and test data saved at artifacts\data_transformation]
[2025-08-04 13:39:52,189: INFO: data_transformation: Train data shape: (1199, 12), Test data shape: (400, 12)]
[2025-08-04 13:39:52,191: INFO: main: >>>>>> stage Data Transformation Stage completed <<<<<<]
[2025-08-04 13:39:52,191: INFO: main: >>>>>> stage Model Training Stage started <<<<<<]
[2025-08-04 13:39:52,193: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-04 13:39:52,199: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-04 13:39:52,203: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-04 13:39:52,206: INFO: common: Created directory at: artifacts]
[2025-08-04 13:39:52,207: INFO: common: Created directory at: artifacts/model_training]
[2025-08-04 13:39:52,279: INFO: main: >>>>>> stage Model Training Stage completed <<<<<<]
[2025-08-04 13:39:52,279: INFO: main: >>>>>> stage Model Evaluation Stage started <<<<<<]
[2025-08-04 13:39:52,287: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-04 13:39:52,289: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-04 13:39:52,292: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-04 13:39:52,294: INFO: common: Created directory at: artifacts]
[2025-08-04 13:39:52,295: INFO: common: Created directory at: artifacts/model_evaluation]
[2025-08-04 13:39:53,371: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-04 13:39:53,403: INFO: helpers: Accessing as rajwaAth]
[2025-08-04 13:39:54,384: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/rajwaAth/MLOps-Project "HTTP/1.1 200 OK"]
[2025-08-04 13:39:55,202: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-04 13:39:55,212: INFO: helpers: Initialized MLflow to track repo "rajwaAth/MLOps-Project"]
[2025-08-04 13:39:55,217: INFO: helpers: Repository rajwaAth/MLOps-Project initialized!]
[2025-08-04 13:39:55,265: INFO: common: JSON file saved at: artifacts\model_evaluation\metrics.json]
[2025-08-04 13:40:20,777: INFO: main: >>>>>> stage Model Evaluation Stage completed <<<<<<]
[2025-08-04 14:15:54,336: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.18.31:8080]
[2025-08-04 14:15:54,337: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-08-04 14:15:54,340: INFO: _internal:  * Restarting with stat]
[2025-08-04 14:15:56,006: WARNING: _internal:  * Debugger is active!]
[2025-08-04 14:15:56,016: INFO: _internal:  * Debugger PIN: 156-117-443]
[2025-08-04 14:16:54,028: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 14:16:54] "GET / HTTP/1.1" 200 -]
[2025-08-04 14:16:54,975: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 14:16:54] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2025-08-04 15:03:27,849: INFO: _internal:  * Detected change in 'c:\\Users\\admin\\OneDrive\\Dokumen\\Portofolio\\Data Sceince (Python)\\End-to-end\\app.py', reloading]
[2025-08-04 15:03:28,102: INFO: _internal:  * Restarting with stat]
[2025-08-04 15:03:30,809: WARNING: _internal:  * Debugger is active!]
[2025-08-04 15:03:30,813: INFO: _internal:  * Debugger PIN: 156-117-443]
[2025-08-04 15:03:31,921: INFO: _internal:  * Detected change in 'c:\\Users\\admin\\OneDrive\\Dokumen\\Portofolio\\Data Sceince (Python)\\End-to-end\\app.py', reloading]
[2025-08-04 15:03:32,141: INFO: _internal:  * Restarting with stat]
[2025-08-04 15:03:33,667: WARNING: _internal:  * Debugger is active!]
[2025-08-04 15:03:33,674: INFO: _internal:  * Debugger PIN: 156-117-443]
[2025-08-04 15:03:38,000: INFO: _internal:  * Detected change in 'c:\\Users\\admin\\OneDrive\\Dokumen\\Portofolio\\Data Sceince (Python)\\End-to-end\\app.py', reloading]
[2025-08-04 15:03:38,216: INFO: _internal:  * Restarting with stat]
[2025-08-04 15:03:39,776: WARNING: _internal:  * Debugger is active!]
[2025-08-04 15:03:39,781: INFO: _internal:  * Debugger PIN: 156-117-443]
[2025-08-04 15:03:45,065: INFO: _internal:  * Detected change in 'c:\\Users\\admin\\OneDrive\\Dokumen\\Portofolio\\Data Sceince (Python)\\End-to-end\\app.py', reloading]
[2025-08-04 15:03:45,368: INFO: _internal:  * Restarting with stat]
[2025-08-04 15:03:49,384: WARNING: _internal:  * Debugger is active!]
[2025-08-04 15:03:49,396: INFO: _internal:  * Debugger PIN: 156-117-443]
[2025-08-04 15:11:53,574: INFO: _internal:  * Detected change in 'c:\\Users\\admin\\OneDrive\\Dokumen\\Portofolio\\Data Sceince (Python)\\End-to-end\\app.py', reloading]
[2025-08-04 15:11:53,801: INFO: _internal:  * Restarting with stat]
[2025-08-04 15:11:55,495: WARNING: _internal:  * Debugger is active!]
[2025-08-04 15:11:55,501: INFO: _internal:  * Debugger PIN: 156-117-443]
[2025-08-04 15:12:11,132: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 15:12:11] "GET / HTTP/1.1" 200 -]
[2025-08-04 15:13:35,174: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 15:13:35] "GET / HTTP/1.1" 200 -]
[2025-08-04 15:16:45,925: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 15:16:45] "GET / HTTP/1.1" 200 -]
[2025-08-04 15:53:12,318: INFO: _internal:  * Detected change in 'c:\\Users\\admin\\OneDrive\\Dokumen\\Portofolio\\Data Sceince (Python)\\End-to-end\\test.py', reloading]
[2025-08-04 15:53:12,855: INFO: _internal:  * Restarting with stat]
[2025-08-04 15:53:15,418: WARNING: _internal:  * Debugger is active!]
[2025-08-04 15:53:15,434: INFO: _internal:  * Debugger PIN: 156-117-443]
[2025-08-04 15:53:17,678: INFO: _internal:  * Detected change in 'c:\\Users\\admin\\OneDrive\\Dokumen\\Portofolio\\Data Sceince (Python)\\End-to-end\\test.py', reloading]
[2025-08-04 15:53:17,861: INFO: _internal:  * Restarting with stat]
[2025-08-04 15:53:19,425: WARNING: _internal:  * Debugger is active!]
[2025-08-04 15:53:19,428: INFO: _internal:  * Debugger PIN: 156-117-443]
[2025-08-04 16:04:01,415: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.18.31:8080]
[2025-08-04 16:04:01,417: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-08-04 16:04:01,420: INFO: _internal:  * Restarting with stat]
[2025-08-04 16:04:03,015: WARNING: _internal:  * Debugger is active!]
[2025-08-04 16:04:03,029: INFO: _internal:  * Debugger PIN: 156-117-443]
[2025-08-04 16:04:05,491: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:04:05] "GET / HTTP/1.1" 200 -]
[2025-08-04 16:04:51,127: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:04:51] "GET / HTTP/1.1" 200 -]
[2025-08-04 16:04:51,177: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:04:51] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -]
[2025-08-04 16:04:51,227: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:04:51] "[33mGET /static/css2/nunito-font.css HTTP/1.1[0m" 404 -]
[2025-08-04 16:04:51,261: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:04:51] "[33mGET /static/css2/style.css HTTP/1.1[0m" 404 -]
[2025-08-04 16:04:51,281: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:04:51] "[33mGET /static/js/scripts.js HTTP/1.1[0m" 404 -]
[2025-08-04 16:04:51,813: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:04:51] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2025-08-04 16:04:51,829: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:04:51] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-08-04 16:04:51,974: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:04:51] "[33mGET /static/assets/favicon.ico HTTP/1.1[0m" 404 -]
[2025-08-04 16:05:37,158: INFO: _internal:  * Detected change in 'C:\\Users\\admin\\OneDrive\\Dokumen\\Portofolio\\Data Sceince (Python)\\End-to-end\\app.py', reloading]
[2025-08-04 16:05:37,386: INFO: _internal:  * Restarting with stat]
[2025-08-04 16:05:39,125: WARNING: _internal:  * Debugger is active!]
[2025-08-04 16:05:39,134: INFO: _internal:  * Debugger PIN: 156-117-443]
[2025-08-04 16:13:02,210: INFO: _internal:  * Detected change in 'C:\\Users\\admin\\OneDrive\\Dokumen\\Portofolio\\Data Sceince (Python)\\End-to-end\\app.py', reloading]
[2025-08-04 16:14:44,526: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.18.31:8080]
[2025-08-04 16:14:44,527: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-08-04 16:14:44,534: INFO: _internal:  * Restarting with stat]
[2025-08-04 16:14:45,784: WARNING: _internal:  * Debugger is active!]
[2025-08-04 16:14:45,785: INFO: _internal:  * Debugger PIN: 156-117-443]
[2025-08-04 16:14:57,135: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:14:57] "GET / HTTP/1.1" 200 -]
[2025-08-04 16:14:57,557: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:14:57] "GET /static/css2/nunito-font.css HTTP/1.1" 200 -]
[2025-08-04 16:14:57,563: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:14:57] "GET /static/css/styles.css HTTP/1.1" 200 -]
[2025-08-04 16:14:57,574: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:14:57] "GET /static/js/scripts.js HTTP/1.1" 200 -]
[2025-08-04 16:14:57,583: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:14:57] "GET /static/css2/style.css HTTP/1.1" 200 -]
[2025-08-04 16:14:57,606: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:14:57] "[33mGET /static/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-08-04 16:14:57,611: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:14:57] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2025-08-04 16:14:57,620: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:14:57] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-08-04 16:15:00,000: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:15:00] "GET / HTTP/1.1" 200 -]
[2025-08-04 16:15:00,058: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:15:00] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2025-08-04 16:15:00,126: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:15:00] "[36mGET /static/css2/nunito-font.css HTTP/1.1[0m" 304 -]
[2025-08-04 16:15:00,174: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:15:00] "[36mGET /static/css2/style.css HTTP/1.1[0m" 304 -]
[2025-08-04 16:15:00,203: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:15:00] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2025-08-04 16:15:00,302: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:15:00] "[33mGET /static/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-08-04 16:15:00,305: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:15:00] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2025-08-04 16:15:00,309: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:15:00] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-08-04 16:15:36,673: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:15:36] "POST /predict HTTP/1.1" 200 -]
[2025-08-04 16:15:36,747: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:15:36] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2025-08-04 16:15:36,797: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:15:36] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2025-08-04 16:15:36,907: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:15:36] "[33mGET /static/assets/img/wine.gif HTTP/1.1[0m" 404 -]
[2025-08-04 16:37:43,698: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.18.31:8080]
[2025-08-04 16:37:43,699: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-08-04 16:37:43,701: INFO: _internal:  * Restarting with stat]
[2025-08-04 16:37:44,873: WARNING: _internal:  * Debugger is active!]
[2025-08-04 16:37:44,876: INFO: _internal:  * Debugger PIN: 156-117-443]
[2025-08-04 16:37:48,088: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:37:48] "GET / HTTP/1.1" 200 -]
[2025-08-04 16:38:01,262: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:38:01] "POST /predict HTTP/1.1" 200 -]
[2025-08-04 16:38:22,029: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:38:22] "GET / HTTP/1.1" 200 -]
[2025-08-04 16:38:27,088: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:38:27] "POST /predict HTTP/1.1" 200 -]
[2025-08-04 16:38:34,908: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:38:34] "GET / HTTP/1.1" 200 -]
[2025-08-04 16:39:09,377: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 16:39:09] "POST /predict HTTP/1.1" 200 -]
[2025-08-04 18:36:55,641: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 18:36:55] "GET / HTTP/1.1" 200 -]
[2025-08-04 18:37:48,409: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 18:37:48] "POST /predict HTTP/1.1" 200 -]
[2025-08-04 18:43:12,801: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.18.31:8080]
[2025-08-04 18:43:12,802: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-08-04 18:43:12,804: INFO: _internal:  * Restarting with stat]
[2025-08-04 18:43:14,278: WARNING: _internal:  * Debugger is active!]
[2025-08-04 18:43:14,280: INFO: _internal:  * Debugger PIN: 156-117-443]
[2025-08-04 18:43:20,550: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 18:43:20] "GET / HTTP/1.1" 200 -]
[2025-08-04 18:43:28,134: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 18:43:28] "POST /predict HTTP/1.1" 200 -]
[2025-08-04 18:51:29,959: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 18:51:29] "GET / HTTP/1.1" 200 -]
[2025-08-04 18:51:31,321: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 18:51:31] "GET / HTTP/1.1" 200 -]
[2025-08-04 18:51:36,327: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 18:51:36] "POST /predict HTTP/1.1" 200 -]
[2025-08-04 18:53:36,147: INFO: _internal:  * Detected change in 'c:\\users\\admin\\onedrive\\dokumen\\portofolio\\data sceince (python)\\end-to-end\\src\\end_to_end_project\\pipeline\\prediction.py', reloading]
[2025-08-04 18:53:36,529: INFO: _internal:  * Restarting with stat]
[2025-08-04 18:53:38,902: WARNING: _internal:  * Debugger is active!]
[2025-08-04 18:53:38,904: INFO: _internal:  * Debugger PIN: 156-117-443]
[2025-08-04 18:53:43,666: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 18:53:43] "POST /predict HTTP/1.1" 200 -]
[2025-08-04 18:53:52,924: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 18:53:52] "GET / HTTP/1.1" 200 -]
[2025-08-04 18:54:22,232: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 18:54:22] "GET / HTTP/1.1" 200 -]
[2025-08-04 18:55:35,997: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 18:55:35] "GET / HTTP/1.1" 200 -]
[2025-08-04 18:58:18,131: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 18:58:18] "GET / HTTP/1.1" 200 -]
[2025-08-04 18:58:20,400: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 18:58:20] "POST /predict HTTP/1.1" 200 -]
[2025-08-04 18:58:25,543: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 18:58:25] "GET / HTTP/1.1" 200 -]
[2025-08-04 18:58:29,296: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 18:58:29] "POST /predict HTTP/1.1" 200 -]
[2025-08-04 18:58:33,709: INFO: _internal: 127.0.0.1 - - [04/Aug/2025 18:58:33] "GET / HTTP/1.1" 200 -]
[2025-08-05 16:39:28,086: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.18.31:8080]
[2025-08-05 16:39:28,089: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-08-05 16:39:28,096: INFO: _internal:  * Restarting with stat]
[2025-08-05 16:39:30,050: WARNING: _internal:  * Debugger is active!]
[2025-08-05 16:39:30,065: INFO: _internal:  * Debugger PIN: 156-117-443]
[2025-08-05 16:39:33,102: INFO: _internal: 127.0.0.1 - - [05/Aug/2025 16:39:33] "GET / HTTP/1.1" 200 -]
[2025-08-05 16:39:43,141: INFO: _internal: 127.0.0.1 - - [05/Aug/2025 16:39:43] "POST /predict HTTP/1.1" 200 -]
[2025-08-08 17:13:17,056: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-08-08 17:13:17,072: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:13:17,090: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:13:17,113: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:13:17,115: INFO: common: Created directory at: artifacts]
[2025-08-08 17:13:17,116: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-08-08 17:13:17,117: INFO: data_ingestion: File already exists of size: 22.7822265625 KB]
[2025-08-08 17:13:17,159: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-08-08 17:13:17,162: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-08-08 17:13:17,162: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-08-08 17:13:17,165: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:13:17,170: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:13:17,176: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:13:17,177: INFO: common: Created directory at: artifacts]
[2025-08-08 17:13:17,180: INFO: common: Created directory at: artifacts/data_validation]
[2025-08-08 17:13:17,226: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-08-08 17:13:17,229: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-08-08 17:13:17,235: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:13:17,246: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:13:17,248: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:13:17,252: INFO: common: Created directory at: artifacts]
[2025-08-08 17:13:17,256: INFO: common: Created directory at: artifacts/data_transformation]
[2025-08-08 17:13:17,258: ERROR: main: DataTransformation.__init__() missing 1 required positional argument: 'df']
Traceback (most recent call last):
  File "c:\Users\admin\OneDrive\Dokumen\Portofolio\Data Sceince (Python)\End-to-end\main.py", line 33, in <module>
    obj.main()
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\pipeline\stage_03_data_transformation.py", line 15, in main
    data_transformation = DataTransformation(config=data_transformation_config)
TypeError: DataTransformation.__init__() missing 1 required positional argument: 'df'
[2025-08-08 17:15:33,836: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-08-08 17:15:33,838: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:15:33,840: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:15:33,844: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:15:33,845: INFO: common: Created directory at: artifacts]
[2025-08-08 17:15:33,847: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-08-08 17:15:33,848: INFO: data_ingestion: File already exists of size: 22.7822265625 KB]
[2025-08-08 17:15:33,851: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-08-08 17:15:33,851: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-08-08 17:15:33,855: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-08-08 17:15:33,861: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:15:33,863: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:15:33,869: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:15:33,870: INFO: common: Created directory at: artifacts]
[2025-08-08 17:15:33,875: INFO: common: Created directory at: artifacts/data_validation]
[2025-08-08 17:15:33,916: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-08-08 17:15:33,917: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-08-08 17:15:33,926: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:15:33,930: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:15:33,933: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:15:33,934: INFO: common: Created directory at: artifacts]
[2025-08-08 17:15:33,935: INFO: common: Created directory at: artifacts/data_transformation]
[2025-08-08 17:15:33,935: ERROR: main: DataTransformation.__init__() missing 1 required positional argument: 'df']
Traceback (most recent call last):
  File "c:\Users\admin\OneDrive\Dokumen\Portofolio\Data Sceince (Python)\End-to-end\main.py", line 33, in <module>
    obj.main()
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\pipeline\stage_03_data_transformation.py", line 15, in main
    data_transformation = DataTransformation(config=data_transformation_config)
TypeError: DataTransformation.__init__() missing 1 required positional argument: 'df'
[2025-08-08 17:19:46,932: INFO: stage_03_data_transformation: >>>>>> stage Data Transformation started <<<<<<]
[2025-08-08 17:19:46,935: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:19:46,935: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:19:46,947: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:19:46,947: INFO: common: Created directory at: artifacts]
[2025-08-08 17:19:46,953: INFO: common: Created directory at: artifacts/data_transformation]
[2025-08-08 17:19:47,024: INFO: data_transformation: Cleaned data shape: (1359, 12)]
[2025-08-08 17:19:47,110: ERROR: stage_03_data_transformation: at least one array or dtype is required]
Traceback (most recent call last):
  File "c:\Users\admin\OneDrive\Dokumen\Portofolio\Data Sceince (Python)\End-to-end\src\end_to_end_project\pipeline\stage_03_data_transformation.py", line 22, in <module>
    obj.main()
  File "c:\Users\admin\OneDrive\Dokumen\Portofolio\Data Sceince (Python)\End-to-end\src\end_to_end_project\pipeline\stage_03_data_transformation.py", line 16, in main
    data_transformation.run_data_transformation()
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\components\data_transformation.py", line 75, in run_data_transformation
    train, test = self.handle_skewness_normalize(train, test)
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\components\data_transformation.py", line 48, in handle_skewness_normalize
    train[skewed_cols] = pt.fit_transform(train[skewed_cols])
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\utils\_set_output.py", line 319, in wrapped
    data_to_wrap = f(self, X, *args, **kwargs)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\base.py", line 1389, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\preprocessing\_data.py", line 3320, in fit_transform
    return self._fit(X, y, force_transform=True)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\preprocessing\_data.py", line 3323, in _fit
    X = self._check_input(X, in_fit=True, check_positive=True)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\preprocessing\_data.py", line 3550, in _check_input
    X = validate_data(
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\utils\validation.py", line 2944, in validate_data
    out = check_array(X, input_name="X", **check_params)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\utils\validation.py", line 931, in check_array
    dtype_orig = np.result_type(*dtypes_orig)
ValueError: at least one array or dtype is required
[2025-08-08 17:20:35,494: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-08-08 17:20:35,496: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:20:35,496: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:20:35,502: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:20:35,503: INFO: common: Created directory at: artifacts]
[2025-08-08 17:20:35,504: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-08-08 17:20:35,505: INFO: data_ingestion: File already exists of size: 22.7822265625 KB]
[2025-08-08 17:20:35,508: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-08-08 17:20:35,508: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-08-08 17:20:35,508: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-08-08 17:20:35,516: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:20:35,518: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:20:35,521: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:20:35,522: INFO: common: Created directory at: artifacts]
[2025-08-08 17:20:35,522: INFO: common: Created directory at: artifacts/data_validation]
[2025-08-08 17:20:35,559: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-08-08 17:20:35,560: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-08-08 17:20:35,565: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:20:35,567: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:20:35,569: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:20:35,570: INFO: common: Created directory at: artifacts]
[2025-08-08 17:20:35,570: INFO: common: Created directory at: artifacts/data_transformation]
[2025-08-08 17:20:35,579: INFO: data_transformation: Cleaned data shape: (1359, 12)]
[2025-08-08 17:20:35,621: ERROR: main: at least one array or dtype is required]
Traceback (most recent call last):
  File "c:\Users\admin\OneDrive\Dokumen\Portofolio\Data Sceince (Python)\End-to-end\main.py", line 33, in <module>
    obj.main()
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\pipeline\stage_03_data_transformation.py", line 16, in main
    data_transformation.run_data_transformation()
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\components\data_transformation.py", line 75, in run_data_transformation
    train, test = self.handle_skewness_normalize(train, test)
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\components\data_transformation.py", line 48, in handle_skewness_normalize
    train[skewed_cols] = pt.fit_transform(train[skewed_cols])
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\utils\_set_output.py", line 319, in wrapped
    data_to_wrap = f(self, X, *args, **kwargs)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\base.py", line 1389, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\preprocessing\_data.py", line 3320, in fit_transform
    return self._fit(X, y, force_transform=True)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\preprocessing\_data.py", line 3323, in _fit
    X = self._check_input(X, in_fit=True, check_positive=True)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\preprocessing\_data.py", line 3550, in _check_input
    X = validate_data(
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\utils\validation.py", line 2944, in validate_data
    out = check_array(X, input_name="X", **check_params)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\utils\validation.py", line 931, in check_array
    dtype_orig = np.result_type(*dtypes_orig)
ValueError: at least one array or dtype is required
[2025-08-08 17:27:58,759: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-08-08 17:27:58,765: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:27:58,767: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:27:58,770: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:27:58,771: INFO: common: Created directory at: artifacts]
[2025-08-08 17:27:58,771: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-08-08 17:27:58,772: INFO: data_ingestion: File already exists of size: 22.7822265625 KB]
[2025-08-08 17:27:58,776: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-08-08 17:27:58,776: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-08-08 17:27:58,777: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-08-08 17:27:58,786: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:27:58,787: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:27:58,795: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:27:58,796: INFO: common: Created directory at: artifacts]
[2025-08-08 17:27:58,796: INFO: common: Created directory at: artifacts/data_validation]
[2025-08-08 17:27:58,830: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-08-08 17:27:58,831: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-08-08 17:27:58,843: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:27:58,845: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:27:58,849: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:27:58,849: INFO: common: Created directory at: artifacts]
[2025-08-08 17:27:58,850: INFO: common: Created directory at: artifacts/data_transformation]
[2025-08-08 17:27:58,860: INFO: data_transformation: Cleaned data shape: (1359, 12)]
[2025-08-08 17:27:58,890: INFO: data_transformation: No skewed columns found, skipping normalization.]
[2025-08-08 17:27:58,912: ERROR: main: Expected n_neighbors <= n_samples_fit, but n_neighbors = 6, n_samples_fit = 2, n_samples = 2]
Traceback (most recent call last):
  File "C:\Users\admin\OneDrive\Dokumen\Portofolio\Data Sceince (Python)\End-to-end\main.py", line 33, in <module>
    obj.main()
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\pipeline\stage_03_data_transformation.py", line 16, in main
    data_transformation.run_data_transformation()
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\components\data_transformation.py", line 80, in run_data_transformation
    train = self.handle_imbalenced(train)
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\components\data_transformation.py", line 61, in handle_imbalenced
    X_resampled, y_resampled = smote.fit_resample(X, y)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\imblearn\base.py", line 202, in fit_resample
    return super().fit_resample(X, y, **params)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\base.py", line 1389, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\imblearn\base.py", line 105, in fit_resample
    output = self._fit_resample(X, y, **params)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\imblearn\over_sampling\_smote\base.py", line 359, in _fit_resample
    nns = self.nn_k_.kneighbors(X_class, return_distance=False)[:, 1:]
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\neighbors\_base.py", line 854, in kneighbors
    raise ValueError(
ValueError: Expected n_neighbors <= n_samples_fit, but n_neighbors = 6, n_samples_fit = 2, n_samples = 2
[2025-08-08 17:32:54,295: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-08-08 17:32:54,300: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:32:54,302: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:32:54,310: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:32:54,312: INFO: common: Created directory at: artifacts]
[2025-08-08 17:32:54,312: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-08-08 17:32:54,313: INFO: data_ingestion: File already exists of size: 22.7822265625 KB]
[2025-08-08 17:32:54,317: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-08-08 17:32:54,319: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-08-08 17:32:54,320: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-08-08 17:32:54,326: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:32:54,328: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:32:54,337: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:32:54,338: INFO: common: Created directory at: artifacts]
[2025-08-08 17:32:54,339: INFO: common: Created directory at: artifacts/data_validation]
[2025-08-08 17:32:54,379: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-08-08 17:32:54,379: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-08-08 17:32:54,386: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:32:54,389: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:32:54,392: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:32:54,393: INFO: common: Created directory at: artifacts]
[2025-08-08 17:32:54,394: INFO: common: Created directory at: artifacts/data_transformation]
[2025-08-08 17:32:54,404: INFO: data_transformation: Cleaned data shape: (1359, 12)]
[2025-08-08 17:32:54,428: INFO: data_transformation: No skewed columns found, skipping normalization.]
[2025-08-08 17:32:54,453: INFO: data_transformation: Minimum sample in any class: 2]
[2025-08-08 17:32:54,457: ERROR: main: Expected n_neighbors <= n_samples_fit, but n_neighbors = 6, n_samples_fit = 2, n_samples = 2]
Traceback (most recent call last):
  File "C:\Users\admin\OneDrive\Dokumen\Portofolio\Data Sceince (Python)\End-to-end\main.py", line 33, in <module>
    obj.main()
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\pipeline\stage_03_data_transformation.py", line 16, in main
    data_transformation.run_data_transformation()
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\components\data_transformation.py", line 85, in run_data_transformation
    train = self.handle_imbalenced(train)
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\components\data_transformation.py", line 66, in handle_imbalenced
    X_resampled, y_resampled = smote.fit_resample(X, y)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\imblearn\base.py", line 202, in fit_resample
    return super().fit_resample(X, y, **params)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\base.py", line 1389, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\imblearn\base.py", line 105, in fit_resample
    output = self._fit_resample(X, y, **params)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\imblearn\over_sampling\_smote\base.py", line 359, in _fit_resample
    nns = self.nn_k_.kneighbors(X_class, return_distance=False)[:, 1:]
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\neighbors\_base.py", line 854, in kneighbors
    raise ValueError(
ValueError: Expected n_neighbors <= n_samples_fit, but n_neighbors = 6, n_samples_fit = 2, n_samples = 2
[2025-08-08 17:34:59,558: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-08-08 17:34:59,561: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:34:59,563: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:34:59,567: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:34:59,568: INFO: common: Created directory at: artifacts]
[2025-08-08 17:34:59,568: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-08-08 17:34:59,569: INFO: data_ingestion: File already exists of size: 22.7822265625 KB]
[2025-08-08 17:34:59,572: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-08-08 17:34:59,573: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-08-08 17:34:59,574: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-08-08 17:34:59,580: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:34:59,582: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:34:59,586: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:34:59,587: INFO: common: Created directory at: artifacts]
[2025-08-08 17:34:59,587: INFO: common: Created directory at: artifacts/data_validation]
[2025-08-08 17:34:59,619: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-08-08 17:34:59,620: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-08-08 17:34:59,628: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:34:59,629: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:34:59,635: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:34:59,636: INFO: common: Created directory at: artifacts]
[2025-08-08 17:34:59,637: INFO: common: Created directory at: artifacts/data_transformation]
[2025-08-08 17:34:59,644: INFO: data_transformation: Cleaned data shape: (1359, 12)]
[2025-08-08 17:34:59,686: INFO: data_transformation: No skewed columns found, skipping normalization.]
[2025-08-08 17:34:59,688: INFO: data_transformation: Minimum sample in any class: 2]
[2025-08-08 17:34:59,693: ERROR: main: SMOTE._fit_resample() got an unexpected keyword argument 'k_neighbors']
Traceback (most recent call last):
  File "C:\Users\admin\OneDrive\Dokumen\Portofolio\Data Sceince (Python)\End-to-end\main.py", line 33, in <module>
    obj.main()
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\pipeline\stage_03_data_transformation.py", line 16, in main
    data_transformation.run_data_transformation()
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\components\data_transformation.py", line 85, in run_data_transformation
    train = self.handle_imbalenced(train)
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\components\data_transformation.py", line 66, in handle_imbalenced
    X_resampled, y_resampled = smote.fit_resample(X, y, k_neighbors=min_samples)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\imblearn\base.py", line 202, in fit_resample
    return super().fit_resample(X, y, **params)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\base.py", line 1389, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\imblearn\base.py", line 105, in fit_resample
    output = self._fit_resample(X, y, **params)
TypeError: SMOTE._fit_resample() got an unexpected keyword argument 'k_neighbors'
[2025-08-08 17:37:18,969: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-08-08 17:37:18,972: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:37:18,975: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:37:18,978: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:37:18,979: INFO: common: Created directory at: artifacts]
[2025-08-08 17:37:18,980: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-08-08 17:37:18,981: INFO: data_ingestion: File already exists of size: 22.7822265625 KB]
[2025-08-08 17:37:18,986: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-08-08 17:37:18,987: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-08-08 17:37:18,988: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-08-08 17:37:18,995: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:37:18,998: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:37:19,001: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:37:19,006: INFO: common: Created directory at: artifacts]
[2025-08-08 17:37:19,008: INFO: common: Created directory at: artifacts/data_validation]
[2025-08-08 17:37:19,039: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-08-08 17:37:19,040: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-08-08 17:37:19,044: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 17:37:19,048: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 17:37:19,051: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 17:37:19,052: INFO: common: Created directory at: artifacts]
[2025-08-08 17:37:19,053: INFO: common: Created directory at: artifacts/data_transformation]
[2025-08-08 17:37:19,057: INFO: data_transformation: Cleaned data shape: (1359, 12)]
[2025-08-08 17:37:19,082: INFO: data_transformation: No skewed columns found, skipping normalization.]
[2025-08-08 17:37:19,092: INFO: data_transformation: Minimum sample in any class: 2]
[2025-08-08 17:37:19,098: ERROR: main: Expected n_neighbors <= n_samples_fit, but n_neighbors = 3, n_samples_fit = 2, n_samples = 2]
Traceback (most recent call last):
  File "C:\Users\admin\OneDrive\Dokumen\Portofolio\Data Sceince (Python)\End-to-end\main.py", line 33, in <module>
    obj.main()
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\pipeline\stage_03_data_transformation.py", line 16, in main
    data_transformation.run_data_transformation()
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\components\data_transformation.py", line 85, in run_data_transformation
    train = self.handle_imbalenced(train)
  File "c:\users\admin\onedrive\dokumen\portofolio\data sceince (python)\end-to-end\src\end_to_end_project\components\data_transformation.py", line 66, in handle_imbalenced
    X_resampled, y_resampled = smote.fit_resample(X, y)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\imblearn\base.py", line 202, in fit_resample
    return super().fit_resample(X, y, **params)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\base.py", line 1389, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\imblearn\base.py", line 105, in fit_resample
    output = self._fit_resample(X, y, **params)
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\imblearn\over_sampling\_smote\base.py", line 359, in _fit_resample
    nns = self.nn_k_.kneighbors(X_class, return_distance=False)[:, 1:]
  File "C:\Users\admin\anaconda3\envs\n2nProject\lib\site-packages\sklearn\neighbors\_base.py", line 854, in kneighbors
    raise ValueError(
ValueError: Expected n_neighbors <= n_samples_fit, but n_neighbors = 3, n_samples_fit = 2, n_samples = 2
[2025-08-08 22:54:44,654: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-08-08 22:54:44,661: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 22:54:44,666: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 22:54:44,669: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 22:54:44,669: INFO: common: Created directory at: artifacts]
[2025-08-08 22:54:44,669: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-08-08 22:54:44,669: INFO: data_ingestion: File already exists of size: 22.7822265625 KB]
[2025-08-08 22:54:44,683: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-08-08 22:54:44,683: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-08-08 22:54:44,683: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-08-08 22:54:44,699: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 22:54:44,707: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 22:54:44,722: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 22:54:44,730: INFO: common: Created directory at: artifacts]
[2025-08-08 22:54:44,730: INFO: common: Created directory at: artifacts/data_validation]
[2025-08-08 22:54:44,789: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-08-08 22:54:44,798: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-08-08 22:54:44,822: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 22:54:44,830: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 22:54:44,838: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 22:54:44,862: INFO: common: Created directory at: artifacts]
[2025-08-08 22:54:44,870: INFO: common: Created directory at: artifacts/data_transformation]
[2025-08-08 22:54:44,934: INFO: data_transformation: Cleaned data shape: (1359, 12)]
[2025-08-08 22:54:44,990: INFO: data_transformation: No skewed columns found, skipping normalization.]
[2025-08-08 22:54:44,990: INFO: data_transformation: Minimum sample in any class: 2]
[2025-08-08 22:54:50,515: INFO: data_transformation: Processed train and test data saved at artifacts\data_transformation]
[2025-08-08 22:54:50,515: INFO: data_transformation: Train data shape: (1974, 12), Test data shape: (340, 12)]
[2025-08-08 22:54:50,515: INFO: main: >>>>>> stage Data Transformation Stage completed <<<<<<]
[2025-08-08 22:54:50,515: INFO: main: >>>>>> stage Model Training Stage started <<<<<<]
[2025-08-08 22:54:50,523: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 22:54:50,523: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 22:54:50,531: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 22:54:50,539: INFO: common: Created directory at: artifacts]
[2025-08-08 22:54:50,539: INFO: common: Created directory at: artifacts/model_training]
[2025-08-08 22:54:50,612: INFO: main: >>>>>> stage Model Training Stage completed <<<<<<]
[2025-08-08 22:54:50,612: INFO: main: >>>>>> stage Model Evaluation Stage started <<<<<<]
[2025-08-08 22:54:50,620: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 22:54:50,620: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 22:54:50,620: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 22:54:50,620: INFO: common: Created directory at: artifacts]
[2025-08-08 22:54:50,620: INFO: common: Created directory at: artifacts/model_evaluation]
[2025-08-08 22:54:51,412: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-08 22:54:51,420: INFO: helpers: Accessing as rajwaAth]
[2025-08-08 22:54:52,283: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/rajwaAth/MLOps-Project "HTTP/1.1 200 OK"]
[2025-08-08 22:54:53,047: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-08 22:54:53,055: INFO: helpers: Initialized MLflow to track repo "rajwaAth/MLOps-Project"]
[2025-08-08 22:54:53,055: INFO: helpers: Repository rajwaAth/MLOps-Project initialized!]
[2025-08-08 22:54:53,088: INFO: common: JSON file saved at: artifacts\model_evaluation\metrics.json]
[2025-08-08 22:55:15,779: INFO: main: >>>>>> stage Model Evaluation Stage completed <<<<<<]
[2025-08-08 23:05:26,069: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-08-08 23:05:26,083: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 23:05:26,093: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 23:05:26,096: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 23:05:26,101: INFO: common: Created directory at: artifacts]
[2025-08-08 23:05:26,102: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-08-08 23:05:26,104: INFO: data_ingestion: File already exists of size: 22.7822265625 KB]
[2025-08-08 23:05:26,120: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-08-08 23:05:26,121: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-08-08 23:05:26,124: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-08-08 23:05:26,131: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 23:05:26,137: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 23:05:26,148: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 23:05:26,149: INFO: common: Created directory at: artifacts]
[2025-08-08 23:05:26,150: INFO: common: Created directory at: artifacts/data_validation]
[2025-08-08 23:05:26,219: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-08-08 23:05:26,219: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-08-08 23:05:26,228: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 23:05:26,233: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 23:05:26,240: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 23:05:26,246: INFO: common: Created directory at: artifacts]
[2025-08-08 23:05:26,247: INFO: common: Created directory at: artifacts/data_transformation]
[2025-08-08 23:05:26,297: INFO: data_transformation: Cleaned data shape: (1359, 12)]
[2025-08-08 23:05:26,361: INFO: data_transformation: No skewed columns found, skipping normalization.]
[2025-08-08 23:05:30,073: INFO: data_transformation: Processed train and test data saved at artifacts\data_transformation]
[2025-08-08 23:05:30,081: INFO: data_transformation: Train data shape: (2574, 12), Test data shape: (340, 12)]
[2025-08-08 23:05:30,082: INFO: main: >>>>>> stage Data Transformation Stage completed <<<<<<]
[2025-08-08 23:05:30,083: INFO: main: >>>>>> stage Model Training Stage started <<<<<<]
[2025-08-08 23:05:30,098: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 23:05:30,100: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 23:05:30,110: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 23:05:30,113: INFO: common: Created directory at: artifacts]
[2025-08-08 23:05:30,114: INFO: common: Created directory at: artifacts/model_training]
[2025-08-08 23:05:30,407: INFO: main: >>>>>> stage Model Training Stage completed <<<<<<]
[2025-08-08 23:05:30,409: INFO: main: >>>>>> stage Model Evaluation Stage started <<<<<<]
[2025-08-08 23:05:30,414: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-08 23:05:30,438: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-08 23:05:30,459: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-08 23:05:30,462: INFO: common: Created directory at: artifacts]
[2025-08-08 23:05:30,466: INFO: common: Created directory at: artifacts/model_evaluation]
[2025-08-08 23:05:31,315: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-08 23:05:31,352: INFO: helpers: Accessing as rajwaAth]
[2025-08-08 23:05:32,092: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/rajwaAth/MLOps-Project "HTTP/1.1 200 OK"]
[2025-08-08 23:05:32,786: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-08 23:05:32,795: INFO: helpers: Initialized MLflow to track repo "rajwaAth/MLOps-Project"]
[2025-08-08 23:05:32,797: INFO: helpers: Repository rajwaAth/MLOps-Project initialized!]
[2025-08-08 23:05:32,843: INFO: common: JSON file saved at: artifacts\model_evaluation\metrics.json]
[2025-08-08 23:05:55,279: INFO: main: >>>>>> stage Model Evaluation Stage completed <<<<<<]
[2025-08-15 14:21:55,352: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-08-15 14:21:55,378: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:21:55,391: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:21:55,401: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:21:55,408: INFO: common: Created directory at: artifacts]
[2025-08-15 14:21:55,409: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-08-15 14:21:55,410: INFO: data_ingestion: File already exists of size: 22.7822265625 KB]
[2025-08-15 14:21:55,439: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-08-15 14:21:55,440: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-08-15 14:21:55,440: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-08-15 14:21:55,447: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:21:55,456: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:21:55,460: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:21:55,462: INFO: common: Created directory at: artifacts]
[2025-08-15 14:21:55,462: INFO: common: Created directory at: artifacts/data_validation]
[2025-08-15 14:21:55,505: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-08-15 14:21:55,506: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-08-15 14:21:55,515: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:21:55,516: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:21:55,519: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:21:55,519: INFO: common: Created directory at: artifacts]
[2025-08-15 14:21:55,519: INFO: common: Created directory at: artifacts/data_transformation]
[2025-08-15 14:21:55,558: INFO: data_transformation: Cleaned data shape: (1359, 12)]
[2025-08-15 14:22:02,011: INFO: data_transformation: Processed train and test data saved at artifacts\data_transformation]
[2025-08-15 14:22:02,014: INFO: data_transformation: Train data shape: (2574, 12), Test data shape: (340, 12)]
[2025-08-15 14:22:02,014: INFO: main: >>>>>> stage Data Transformation Stage completed <<<<<<]
[2025-08-15 14:22:02,015: INFO: main: >>>>>> stage Model Training Stage started <<<<<<]
[2025-08-15 14:22:02,017: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:22:02,020: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:22:02,024: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:22:02,028: INFO: common: Created directory at: artifacts]
[2025-08-15 14:22:02,029: INFO: common: Created directory at: artifacts/model_training]
[2025-08-15 14:22:02,114: INFO: main: >>>>>> stage Model Training Stage completed <<<<<<]
[2025-08-15 14:22:02,115: INFO: main: >>>>>> stage Model Evaluation Stage started <<<<<<]
[2025-08-15 14:22:02,121: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:22:02,123: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:22:02,130: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:22:02,132: INFO: common: Created directory at: artifacts]
[2025-08-15 14:22:02,137: INFO: common: Created directory at: artifacts/model_evaluation]
[2025-08-15 14:22:02,949: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-15 14:22:02,997: INFO: helpers: Accessing as rajwaAth]
[2025-08-15 14:22:03,765: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/rajwaAth/MLOps-Project "HTTP/1.1 200 OK"]
[2025-08-15 14:22:04,481: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-15 14:22:04,502: INFO: helpers: Initialized MLflow to track repo "rajwaAth/MLOps-Project"]
[2025-08-15 14:22:04,507: INFO: helpers: Repository rajwaAth/MLOps-Project initialized!]
[2025-08-15 14:22:04,581: INFO: common: JSON file saved at: artifacts\model_evaluation\metrics.json]
[2025-08-15 14:22:29,155: INFO: main: >>>>>> stage Model Evaluation Stage completed <<<<<<]
[2025-08-15 14:25:28,490: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-08-15 14:25:28,493: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:25:28,503: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:25:28,507: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:25:28,510: INFO: common: Created directory at: artifacts]
[2025-08-15 14:25:28,510: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-08-15 14:25:28,511: INFO: data_ingestion: File already exists of size: 22.7822265625 KB]
[2025-08-15 14:25:28,518: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-08-15 14:25:28,520: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-08-15 14:25:28,521: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-08-15 14:25:28,528: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:25:28,530: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:25:28,531: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:25:28,539: INFO: common: Created directory at: artifacts]
[2025-08-15 14:25:28,541: INFO: common: Created directory at: artifacts/data_validation]
[2025-08-15 14:25:28,587: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-08-15 14:25:28,589: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-08-15 14:25:28,594: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:25:28,598: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:25:28,603: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:25:28,605: INFO: common: Created directory at: artifacts]
[2025-08-15 14:25:28,606: INFO: common: Created directory at: artifacts/data_transformation]
[2025-08-15 14:25:28,622: INFO: data_transformation: Cleaned data shape: (1359, 12)]
[2025-08-15 14:25:33,164: INFO: data_transformation: Processed train and test data saved at artifacts\data_transformation]
[2025-08-15 14:25:33,165: INFO: data_transformation: Train data shape: (2574, 12), Test data shape: (340, 12)]
[2025-08-15 14:25:33,165: INFO: main: >>>>>> stage Data Transformation Stage completed <<<<<<]
[2025-08-15 14:25:33,166: INFO: main: >>>>>> stage Model Training Stage started <<<<<<]
[2025-08-15 14:25:33,167: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:25:33,172: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:25:33,177: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:25:33,179: INFO: common: Created directory at: artifacts]
[2025-08-15 14:25:33,180: INFO: common: Created directory at: artifacts/model_training]
[2025-08-15 14:25:33,255: INFO: main: >>>>>> stage Model Training Stage completed <<<<<<]
[2025-08-15 14:25:33,255: INFO: main: >>>>>> stage Model Evaluation Stage started <<<<<<]
[2025-08-15 14:25:33,260: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:25:33,264: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:25:33,268: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:25:33,272: INFO: common: Created directory at: artifacts]
[2025-08-15 14:25:33,273: INFO: common: Created directory at: artifacts/model_evaluation]
[2025-08-15 14:25:33,993: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-15 14:25:34,018: INFO: helpers: Accessing as rajwaAth]
[2025-08-15 14:25:34,762: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/rajwaAth/MLOps-Project "HTTP/1.1 200 OK"]
[2025-08-15 14:25:35,534: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-15 14:25:35,545: INFO: helpers: Initialized MLflow to track repo "rajwaAth/MLOps-Project"]
[2025-08-15 14:25:35,555: INFO: helpers: Repository rajwaAth/MLOps-Project initialized!]
[2025-08-15 14:25:35,579: INFO: common: JSON file saved at: artifacts\model_evaluation\metrics.json]
[2025-08-15 14:25:53,950: INFO: main: >>>>>> stage Model Evaluation Stage completed <<<<<<]
[2025-08-15 14:26:28,836: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-08-15 14:26:28,839: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:26:28,844: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:26:28,848: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:26:28,850: INFO: common: Created directory at: artifacts]
[2025-08-15 14:26:28,851: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-08-15 14:26:28,851: INFO: data_ingestion: File already exists of size: 22.7822265625 KB]
[2025-08-15 14:26:28,859: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-08-15 14:26:28,860: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-08-15 14:26:28,861: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-08-15 14:26:28,865: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:26:28,877: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:26:28,885: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:26:28,887: INFO: common: Created directory at: artifacts]
[2025-08-15 14:26:28,889: INFO: common: Created directory at: artifacts/data_validation]
[2025-08-15 14:26:28,930: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-08-15 14:26:28,930: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-08-15 14:26:28,940: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:26:28,946: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:26:28,951: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:26:28,955: INFO: common: Created directory at: artifacts]
[2025-08-15 14:26:28,956: INFO: common: Created directory at: artifacts/data_transformation]
[2025-08-15 14:26:28,973: INFO: data_transformation: Cleaned data shape: (1359, 12)]
[2025-08-15 14:26:29,005: INFO: data_transformation: Processed train and test data saved at artifacts\data_transformation]
[2025-08-15 14:26:29,005: INFO: data_transformation: Train data shape: (1019, 12), Test data shape: (340, 12)]
[2025-08-15 14:26:29,006: INFO: main: >>>>>> stage Data Transformation Stage completed <<<<<<]
[2025-08-15 14:26:29,007: INFO: main: >>>>>> stage Model Training Stage started <<<<<<]
[2025-08-15 14:26:29,012: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:26:29,015: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:26:29,017: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:26:29,023: INFO: common: Created directory at: artifacts]
[2025-08-15 14:26:29,025: INFO: common: Created directory at: artifacts/model_training]
[2025-08-15 14:26:29,085: INFO: main: >>>>>> stage Model Training Stage completed <<<<<<]
[2025-08-15 14:26:29,085: INFO: main: >>>>>> stage Model Evaluation Stage started <<<<<<]
[2025-08-15 14:26:29,092: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:26:29,094: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:26:29,100: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:26:29,102: INFO: common: Created directory at: artifacts]
[2025-08-15 14:26:29,103: INFO: common: Created directory at: artifacts/model_evaluation]
[2025-08-15 14:26:29,804: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-15 14:26:29,817: INFO: helpers: Accessing as rajwaAth]
[2025-08-15 14:26:30,502: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/rajwaAth/MLOps-Project "HTTP/1.1 200 OK"]
[2025-08-15 14:26:31,341: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-15 14:26:31,346: INFO: helpers: Initialized MLflow to track repo "rajwaAth/MLOps-Project"]
[2025-08-15 14:26:31,347: INFO: helpers: Repository rajwaAth/MLOps-Project initialized!]
[2025-08-15 14:26:31,370: INFO: common: JSON file saved at: artifacts\model_evaluation\metrics.json]
[2025-08-15 14:26:50,593: INFO: main: >>>>>> stage Model Evaluation Stage completed <<<<<<]
[2025-08-15 14:28:57,395: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-08-15 14:28:57,400: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:28:57,401: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:28:57,407: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:28:57,408: INFO: common: Created directory at: artifacts]
[2025-08-15 14:28:57,409: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-08-15 14:28:57,409: INFO: data_ingestion: File already exists of size: 22.7822265625 KB]
[2025-08-15 14:28:57,418: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-08-15 14:28:57,423: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-08-15 14:28:57,424: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-08-15 14:28:57,432: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:28:57,436: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:28:57,441: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:28:57,447: INFO: common: Created directory at: artifacts]
[2025-08-15 14:28:57,448: INFO: common: Created directory at: artifacts/data_validation]
[2025-08-15 14:28:57,495: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-08-15 14:28:57,497: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-08-15 14:28:57,507: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:28:57,509: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:28:57,513: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:28:57,513: INFO: common: Created directory at: artifacts]
[2025-08-15 14:28:57,514: INFO: common: Created directory at: artifacts/data_transformation]
[2025-08-15 14:28:57,545: INFO: data_transformation: Processed train and test data saved at artifacts\data_transformation]
[2025-08-15 14:28:57,545: INFO: data_transformation: Train data shape: (1199, 12), Test data shape: (400, 12)]
[2025-08-15 14:28:57,545: INFO: main: >>>>>> stage Data Transformation Stage completed <<<<<<]
[2025-08-15 14:28:57,546: INFO: main: >>>>>> stage Model Training Stage started <<<<<<]
[2025-08-15 14:28:57,547: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:28:57,552: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:28:57,557: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:28:57,559: INFO: common: Created directory at: artifacts]
[2025-08-15 14:28:57,559: INFO: common: Created directory at: artifacts/model_training]
[2025-08-15 14:28:57,608: INFO: main: >>>>>> stage Model Training Stage completed <<<<<<]
[2025-08-15 14:28:57,612: INFO: main: >>>>>> stage Model Evaluation Stage started <<<<<<]
[2025-08-15 14:28:57,615: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:28:57,618: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:28:57,620: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:28:57,621: INFO: common: Created directory at: artifacts]
[2025-08-15 14:28:57,623: INFO: common: Created directory at: artifacts/model_evaluation]
[2025-08-15 14:28:58,388: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-15 14:28:58,427: INFO: helpers: Accessing as rajwaAth]
[2025-08-15 14:28:59,310: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/rajwaAth/MLOps-Project "HTTP/1.1 200 OK"]
[2025-08-15 14:29:00,435: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-15 14:29:00,444: INFO: helpers: Initialized MLflow to track repo "rajwaAth/MLOps-Project"]
[2025-08-15 14:29:00,461: INFO: helpers: Repository rajwaAth/MLOps-Project initialized!]
[2025-08-15 14:29:00,543: INFO: common: JSON file saved at: artifacts\model_evaluation\metrics.json]
[2025-08-15 14:29:22,260: INFO: main: >>>>>> stage Model Evaluation Stage completed <<<<<<]
[2025-08-15 14:29:41,790: INFO: stage_03_data_transformation: >>>>>> stage Data Transformation started <<<<<<]
[2025-08-15 14:29:41,792: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:29:41,795: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:29:41,799: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:29:41,801: INFO: common: Created directory at: artifacts]
[2025-08-15 14:29:41,803: INFO: common: Created directory at: artifacts/data_transformation]
[2025-08-15 14:29:41,812: INFO: data_transformation: Cleaned data shape: (1359, 12)]
[2025-08-15 14:29:41,842: INFO: data_transformation: Processed train and test data saved at artifacts\data_transformation]
[2025-08-15 14:29:41,845: INFO: data_transformation: Train data shape: (1019, 12), Test data shape: (340, 12)]
[2025-08-15 14:29:41,846: INFO: stage_03_data_transformation: >>>>>> stage Data Transformation completed <<<<<<]
[2025-08-15 14:29:55,498: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-08-15 14:29:55,501: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:29:55,504: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:29:55,506: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:29:55,508: INFO: common: Created directory at: artifacts]
[2025-08-15 14:29:55,508: INFO: common: Created directory at: artifacts/data_ingestion]
[2025-08-15 14:29:55,509: INFO: data_ingestion: File already exists of size: 22.7822265625 KB]
[2025-08-15 14:29:55,513: INFO: data_ingestion: Extracted zip file to : artifacts/data_ingestion]
[2025-08-15 14:29:55,516: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<]
[2025-08-15 14:29:55,517: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-08-15 14:29:55,521: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:29:55,527: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:29:55,534: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:29:55,536: INFO: common: Created directory at: artifacts]
[2025-08-15 14:29:55,537: INFO: common: Created directory at: artifacts/data_validation]
[2025-08-15 14:29:55,590: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<]
[2025-08-15 14:29:55,590: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-08-15 14:29:55,595: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:29:55,597: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:29:55,603: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:29:55,603: INFO: common: Created directory at: artifacts]
[2025-08-15 14:29:55,605: INFO: common: Created directory at: artifacts/data_transformation]
[2025-08-15 14:29:55,623: INFO: data_transformation: Cleaned data shape: (1359, 12)]
[2025-08-15 14:29:55,651: INFO: data_transformation: Processed train and test data saved at artifacts\data_transformation]
[2025-08-15 14:29:55,652: INFO: data_transformation: Train data shape: (1019, 12), Test data shape: (340, 12)]
[2025-08-15 14:29:55,652: INFO: main: >>>>>> stage Data Transformation Stage completed <<<<<<]
[2025-08-15 14:29:55,653: INFO: main: >>>>>> stage Model Training Stage started <<<<<<]
[2025-08-15 14:29:55,657: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:29:55,660: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:29:55,663: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:29:55,664: INFO: common: Created directory at: artifacts]
[2025-08-15 14:29:55,666: INFO: common: Created directory at: artifacts/model_training]
[2025-08-15 14:29:55,732: INFO: main: >>>>>> stage Model Training Stage completed <<<<<<]
[2025-08-15 14:29:55,734: INFO: main: >>>>>> stage Model Evaluation Stage started <<<<<<]
[2025-08-15 14:29:55,738: INFO: common: YAML file: config\config.yaml loaded successfully]
[2025-08-15 14:29:55,742: INFO: common: YAML file: params.yaml loaded successfully]
[2025-08-15 14:29:55,744: INFO: common: YAML file: schema.yaml loaded successfully]
[2025-08-15 14:29:55,746: INFO: common: Created directory at: artifacts]
[2025-08-15 14:29:55,747: INFO: common: Created directory at: artifacts/model_evaluation]
[2025-08-15 14:29:56,454: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-15 14:29:56,485: INFO: helpers: Accessing as rajwaAth]
[2025-08-15 14:29:57,257: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/rajwaAth/MLOps-Project "HTTP/1.1 200 OK"]
[2025-08-15 14:29:57,988: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-08-15 14:29:58,006: INFO: helpers: Initialized MLflow to track repo "rajwaAth/MLOps-Project"]
[2025-08-15 14:29:58,011: INFO: helpers: Repository rajwaAth/MLOps-Project initialized!]
[2025-08-15 14:29:58,054: INFO: common: JSON file saved at: artifacts\model_evaluation\metrics.json]
[2025-08-15 14:30:17,747: INFO: main: >>>>>> stage Model Evaluation Stage completed <<<<<<]
